transcendence-app/src/main.ts

```diff
@@ -22,8 +24,13 @@ async function bootstrap() {
     .setVersion('1.0')
     .build();
   const document = SwaggerModule.createDocument(app, options);
-  const yamlString: string = yaml.stringify(document, {});
-  writeFileSync('./swagger-spec.yaml', yamlString);
+  const configService =
+    app.get<ConfigService<EnvironmentVariables>>(ConfigService);
+  const nodeEnv = configService.get<string>('NODE_ENV');
+  if (nodeEnv && nodeEnv !== 'production') {
+    const yamlString: string = yaml.stringify(document, {});
+    writeFileSync('./swagger-spec.yaml', yamlString);
+  }
```

Comment:
There was some permission error while creating this file in production environment.

If we want to keep it, we could change the permissions for the /transcendence-app/dist directory.

Reply:
I've removed this and fixed the directory permission.